packages/@sanity/form-builder/src/inputs/arrays/PortableText/PortableTextInput.tsx

```diff
@@ -12,15 +12,16 @@ import {
   PortableTextEditor,
   Type,
   HotkeyOptions,
+  InvalidValue,
 } from '@sanity/portable-text-editor'
 import {Subject} from 'rxjs'
-import {useToast} from '@sanity/ui'
+import {Box, useToast} from '@sanity/ui'
 import PatchEvent from '../../../PatchEvent'
 import withPatchSubscriber from '../../../utils/withPatchSubscriber'
 import type {Patch} from '../../../patch/types'
 import {RenderBlockActions, RenderCustomMarkers} from './types'
 import Input from './Input'
-import RespondToInvalidContent from './InvalidValue'
+import {InvalidValue as RespondToInvalidContent} from './InvalidValue'
```

Comment:
A bit unclear to me why this is renamed here

Reply:
The name is clashing with another variable